test/javascript/test_market_makers.js

```diff
@@ -53,44 +54,45 @@ contract('MarketMaker', function(accounts) {
-        await market.fund(funding, { from: accounts[investor] })
+        await etherToken.approve(market.address, funding.toString(), { from: accounts[investor] })
+        await market.fund(funding.toString(), { from: accounts[investor] })
         assert.equal(await etherToken.balanceOf.call(accounts[investor]), 0)
 
         // User buys all outcomes
         const trader = 1
         const outcome = 1
-        const outcomeToken = OutcomeToken.at(await event.outcomeTokens.call(outcome))
+        const outcomeToken = await OutcomeToken.at(await event.outcomeTokens.call(outcome))
         const tokenCount = 1e18
         const loopCount = 10
 
         await etherToken.deposit({ value: tokenCount * loopCount, from: accounts[trader] })
-        await etherToken.approve(event.address, tokenCount * loopCount, { from: accounts[trader] })
-        await event.buyAllOutcomes(tokenCount * loopCount, { from: accounts[trader] })
+        await etherToken.approve(event.address, (tokenCount * loopCount).toString(), { from: accounts[trader] })
+        await event.buyAllOutcomes((tokenCount * loopCount).toString(), { from: accounts[trader] })
 
         // User sells tokens
         const buyerBalance = await etherToken.balanceOf.call(accounts[trader])
         let profit, outcomeTokenAmounts
         for(let i of _.range(loopCount)) {
             // Calculate profit for selling tokens
-            outcomeTokenAmounts = Array.from({length: numOutcomes}, (v, i) => i === outcome ? -tokenCount : 0)
+            outcomeTokenAmounts = Array.from({length: numOutcomes}, (v, i) => i === outcome ? (-tokenCount).toString() : 0)
             profit = (await lmsrMarketMaker.calcNetCost.call(market.address, outcomeTokenAmounts)).neg()
             if(profit == 0)
                 break
 
             // Selling tokens
-            await outcomeToken.approve(market.address, tokenCount, { from: accounts[trader] })
-            assert.equal(getParamFromTxEvent(
+            await outcomeToken.approve(market.address, tokenCount.toString(), { from: accounts[trader] })
+            assert.equal((await getParamFromTxEvent(
                 await market.trade(outcomeTokenAmounts, profit.neg(), { from: accounts[trader] }), 'outcomeTokenNetCost'
-            ).neg().valueOf(), profit.valueOf())
+            )).neg().valueOf(), profit.toString())
 
             let netOutcomeTokensSold = await Promise.all(_.range(numOutcomes).map((j) => market.netOutcomeTokensSold(j)))
             let expected = lmsrMarginalPrice(funding, netOutcomeTokensSold, outcome)
-            let actual = (await lmsrMarketMaker.calcMarginalPrice.call(market.address, outcome)).div(ONE)
+            let actual = new BigNumber((await lmsrMarketMaker.calcMarginalPrice.call(market.address, outcome.toString())))
+                .div(ONE.toString())
             assert(
-                isClose(actual, expected),
+                isClose(actual, expected, relTol=1e-3, absTol=1e-9),
```

Comment:
I think this will actually create `relTol` and `absTol` as global variables and set them to those values. I believe what you actually want to do is just:

```js
isClose(actual, expected, 1e-3, 1e-9)
```

Reply:
Thanks for your appreciation.
I updated the code accordingly. 
Regarding the version, do you like me to update the version at `package.json` to be `1.3.0`? Anything else do I have to change regarding?
Many thanks

Reply:
No, I can take care of the versioning later. Thanks!